sql/src/main/java/io/crate/execution/ddl/tables/AlterTableOperation.java

```diff
@@ -464,20 +464,17 @@ private static DiscoveryNode getNodeForResize(ClusterState state) {
                         request.ackTimeout(),
                         shardsAcked -> {
                             if (!shardsAcked) {
-                                logger.warn("[{}] indexes shrinked, but the operation timed out while waiting for " +
-                                            "enough shards to be started. Timeout={}. " +
-                                            "Consider increasing the timeout or adding nodes to the cluster and retry the operation",
-                                    request.getTargetIndexRequest().indices(), request.timeout());
-                                throw new RuntimeException("Shrinking Request failed");
+                                throw new RuntimeException("Shrinking operation was successful but the operation " +
+                                                           "timed out while waiting for enough shards to be started. " +
+                                                           "Consider increasing the timeout and then retry the operation");
```

Comment:
This doesn't make sense. The user can't increase the timeout and there is no point in retrying the operation. The shards will become active without user intervention (unless there are disk space issues or too few nodes, in which case the user has to resolve these issues instead of retrying the operation)